lib/bounces.js

```diff
@@ -120,14 +120,7 @@ module.exports = function (log, error) {
 
           // Flow events have a specific event and structure that must be emitted.
           // Ref `gather` in https://github.com/mozilla/fxa-auth-server/blob/master/lib/metrics/context.js
-          const flowEventInfo = {
-            event: eventName,
-            time: currentTime,
-            flow_id: flowId,
-            flow_time: elapsedTime
-          }
-
-          log.flowEvent(flowEventInfo)
```

Comment:
Oh didn't realize that this got refactored. My understanding is the way it is being logged in train-77 is not correct because it is not being capture by import script. @philbooth would differences in auth-server and bouncer process contribute to some of the discrepancy in log outputs?